src/compute/src/sink/correction.rs

```diff
@@ -47,17 +47,24 @@ pub(super) struct Correction<D> {
     metrics: SinkMetrics,
     /// Per-worker persist sink metrics.
     worker_metrics: SinkWorkerMetrics,
+    /// Configuration for `ConsolidatingVec` determining the rate of growth (doubling, or less).
+    growth_denominator: usize,
```

Comment:
There isn't anything that enforces "doubling or less" here, right? People could set the config to `1` or `0` and end up having a bad time. How about checking that in `new` and falling back to `2` on invalid values? Alternatively, redefine the semantics to have `0` mean doubling.

Reply:
Nope. Good point. This might be a great change also because `2` only coincidentally means "doubling", and perhaps we should break that association.